### PR TITLE
Render median base, extra, and total pay

### LINKED ISSUE #367: Calculate employer pay medians
Since we've broken out base and extra pay, our [current strategy](https://github.com/datamade/bga-payroll/blob/b94988345669c7ee8e950913e4418580b0ea62e8/payroll/views.py#L105) for calculating median total pay will now calculate median base salary. 

- [x] Refactor [the median base salary method](https://github.com/datamade/bga-payroll/blob/b94988345669c7ee8e950913e4418580b0ea62e8/payroll/views.py#L105) to start from the Employer, rather than the Salaries. This will look similar [to the employer payroll calculation](https://github.com/datamade/bga-payroll/blob/1e19f4cdf720174a09392db6dc3681b0f23c0d6c/payroll/views.py#L97).
- [x] Using the same pattern, calculate median extra pay and median total pay. Add these figures to the template context.
- [x] Update the templates to display the calculated values.

### PR DESCRIPTION
**Pull request by fatima3558**
### Overview
This PR calculates median base, extra, and total pay and renders these stats on the relevant cards on the unit and department pages.

Closes #367 
Closes #374

### CODE REVIEW

File: payroll/views.py [hunk 1]
```diff
@@ -141,14 +147,34 @@ def get_context_data(self, **kwargs):
...
+            Q(id=self.object.id) | Q(parent_id=self.object.id)
+        )
+
+        median_base_pay = Percentile(
+            "positions__jobs__salaries__amount", 0.5, output_field=FloatField()
+        )
+        median_extra_pay = Percentile(
+            "positions__jobs__salaries__extra_pay", 0.5, output_field=FloatField()
+        )
+        median_total_pay = Percentile(
```
Comment (hancush) [inline payroll/views.py]: There's a subtle issue with this approach that I didn't anticipate. Let's talk it through IRL on Monday morning.

Comment (hancush) [reply payroll/views.py]: The good news: It's fixable with a tiny change.

**Comment by fatima3558** [comment]
I realized I made a mistake with my use of `NullIf`. Sorry for the premature PR request!

File: templates/partials/baseball_stats.html [hunk 2]
```diff
@@ -18,7 +18,7 @@ <h3>${{ total_expenditure|format_ballpark_number }}</h3>
 </div>
 <div class="col-md-3 col-6">
   <div class="card-body text-center">
-    <h3>{{ median_salary|format_salary }}</h3>
-    <small class="text-muted">Median salary</small>
+    <h3>{{ median_tp|format_salary }}</h3>
+    <small class="text-muted">Median Total Salary</small>
```
Comment (hancush) [inline templates/partials/baseball_stats.html]: Copy editing nitpick: I think of salary as a fixed payment on a set schedule, i.e., appropriate to use for base pay, but probably not for figures including variable extra pay. How about "median total pay"?

Comment (hancush) [reply templates/partials/baseball_stats.html]: (Here and in the `payroll_expenditure.html` partial.)

**Pull request merged**